app/src/main/java/to/dev/dev_android/view/main/view/MainActivity.kt

```diff
@@ -60,15 +60,16 @@ class MainActivity : BaseActivity<ActivityMainBinding>(), CustomWebChromeClient.
     }
 
     override fun launchGallery(filePathCallback: ValueCallback<Array<Uri>>?) {
-        mFilePathCallback = filePathCallback
+        this.filePathCallback = filePathCallback
 
-        val galleryIntent = Intent()
-        // Show only images, no videos or anything else
-        galleryIntent.type = "image/*"
-        galleryIntent.action = Intent.ACTION_PICK
+        val galleryIntent = Intent().apply {
+            // Show only images, no videos or anything else
+            type = "image/*"
+            action = Intent.ACTION_PICK
+        }
 
         // Always show the chooser (if there are multiple options available)
-        ActivityCompat.startActivityForResult(
+        startActivityForResult(
```

Comment:
It won't be clear to someone reading the code as to whether the `startActivityForResult` is the one from the parent class of `MainActivity` or from `ActivityCompat` (as is the case here).
It would be better to keep it verbose in this case and go with `ActivityCompay.startActivityForResult`

Reply:
Internally the `startActivityForResult` will use the one from the activity passed as parameter and the need of this method was to validate the SDK version.

This is the body of `ActivityCompat.startActivityForResult`:

```java
if (Build.VERSION.SDK_INT >= 16) {
    activity.startActivityForResult(intent, requestCode, options);
} else {
    activity.startActivityForResult(intent, requestCode);
}
```

And because we are using a minSDK of 18 and target is 28 I think we should remove this call.

Reply:
That sounds acceptable.
To infinity and beyond 🚀